docs/dashboards/introduction.md

```diff
@@ -28,10 +28,21 @@ In the top right of the screen, click the **+ New** > **Dashboard**. Give your n
 
 ## Adding questions to a dashboard
 
-There are two ways to add questions to a dashboard: from the dashboard, or from the question you want to add.
+There are different ways to add questions to a dashboard:
 
-- **From a question**: You can add a newly saved question to a dashboard directly from the modal that pops up after you save the question for the first time. You can also add a question to a dashboard by clicking on the ellipsis (**...**) at the top right of a question and selecting **Add to dashboard**.
-- **From a dashboard**: Click on the **pencil** icon to **edit the dashboard**. Then click the **+** icon in the top right of the dashboard editing interface (not the **+** in the main navigation bar) to add any of your saved questions to the dashboard, regardless of which collection the questions are in.
+### Saving questions to a dashboard
```

Comment:
Another section on questions in dashboards.

Reply:
One of the main changes of this project is that now people can create questions when in the dashboard edit mode.

This section explains how to add a question from a dashboard, but it isn't clear that you can create a new question